usr/src/boot/sys/boot/i386/loader/loader.rc

```diff
@@ -8,6 +8,10 @@ include /boot/forth/loader.4th
 set tem.fg_color=white
 set tem.bg_color=black
 
+\ As another SmartOS customization, work around OS-8160 by limiting the size of
+\ the framebuffer.
+framebuffer set 1024x768
```

Comment:
What about old monitors that are <=800x600, if it doesn't work... do we care about those?

Reply:
This is really only there in the first place for recovery purposes on test systems that lack serial console. I don't think it makes sense to be worrying ourselves about ancient monitors.